bumble/transport/ws_client.py

```diff
@@ -27,7 +27,7 @@
 
 
 # -----------------------------------------------------------------------------
-async def open_ws_client_transport(spec):
+async def open_ws_client_transport(spec: str) -> PumpedTransport:
```

Comment:
Prefer returning just `Transport` to avoid leaking internal details.

Reply:
Done